tests/Microsoft.Bot.Builder.Dialogs.Adaptive.Tests/Tests/TestScriptTests/PropertyMock/settings.mock.dialog

```diff
@@ -0,0 +1,26 @@
+﻿{
+    "$schema": "../../../../tests.schema",
+    "$kind": "Microsoft.Test.SettingsPropertiesMock",
+    "assignments": [
+        {
+            "property": "fileoverwrite",
```

Comment:
>fileoverwrite [](start = 25, length = 13)

These can just be "settings.fileoverwrite"

Reply:
All updated